Kiosk/App/GlobalFunctions.swift

```diff
@@ -31,3 +31,28 @@ func responseIsOK(object: AnyObject!) -> AnyObject {
     }
     return false
 }
+
+// Adapted from https://github.com/FUKUZAWA-Tadashi/FHCCommander/blob/67c67757ee418a106e0ce0c0820459299b3d77bb/fhcc/Convenience.swift#L33-L44
+func getSSID() -> String? {
+    let interfaces: CFArray! = CNCopySupportedInterfaces()
+    if interfaces == nil { return nil }
+
+    let if0: UnsafePointer<Void>? = CFArrayGetValueAtIndex(interfaces, 0)
+    if if0 == nil { return nil }
+
+    let interfaceName: CFStringRef = unsafeBitCast(if0!, CFStringRef.self)
+    let dictionary = CNCopyCurrentNetworkInfo(interfaceName) as NSDictionary?
+    if dictionary == nil { return nil }
+
+    return dictionary?[kCNNetworkInfoKeySSID as String] as? String
+}
+
+func detectDevelopment() -> Bool {
```

Comment:
I'm not sure that this function name is entirely descriptive, `detectDevelopmentEnvironment` ?